### PR TITLE
Redirecting user to login page when logged out.

### PR DESCRIPTION
**Pull request by gwthm-in**
Dashboard or Welcome pages are keep rendering when user logged out. Redirecting them to login.

### CODE REVIEW

**Comment by bkyryliuk** [comment]
dub of: 
https://github.com/airbnb/superset/pull/1723

**Comment by kkalyan** [comment]
does this work for you? It does not work for me.

**Comment by Gowtham95india** [comment]
@kkalyan, yes it is working.

File: superset/views.py [hunk 1]
```diff
@@ -2777,6 +2777,7 @@ def show_traceback(self):
             error_msg=get_error_msg(),
         ), 500
 
+    @has_access  # Gowtham95india - Login issue when logged out.
```
Comment (xrmx) [inline superset/views.py]: Remove that kind of comments please, we have git for tracking history

**Comment by mistercrunch** [comment]
Looks like this breaks the build. Please fix the unit tests.

**Pull request closed without merging**